core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -118,8 +118,8 @@ public int minWordLength() {
         return minWordLength;
     }
 
-    public void minQueryLength(int minQueryLength) {
-        this.minWordLength = minQueryLength;
+    public void minWordLength(int minWordLength) {
```

Comment:
Thanks for pointing that out, I didn't know of that file.  That begs the question, do you think such a change is worth having an API incompatibility change?  `minWordLength` I feel is more accurate and more consistent, but is it worth breaking backward compatibility?  Or should we keep `minQueryLength` in addition to `minWordLength` and add a deprecated annotation for `minQueryLength`?